### PR TITLE
Doxygen fixes

### PR DESCRIPTION
**Pull request by amiclaus**
Fixes related to the TravisCI Doxygen build.

### CODE REVIEW

File: drivers/rf-transceiver/ad9361/ad9361.c [hunk 1]
```diff
@@ -484,41 +484,42 @@ static const uint8_t full_gain_table[RXGAIN_TBLS_END][SIZE_FULL_TABLE][3] = {
 	}
 };
 
-static const int8_t full_gain_table_abs_gain[RXGAIN_TBLS_END][SIZE_FULL_TABLE] = {
+static const int8_t full_gain_table_abs_gain[RXGAIN_TBLS_END][SIZE_FULL_TABLE]
+= {
 	{  /* 800 MHz */
 		-1, -1, -1, 0, 1, 2, 3, 4,
```
Comment (scuciurean) [inline drivers/rf-transceiver/ad9361/ad9361.c:490]: Why not fix this line too? In  raw mode it still looks bad

Comment (amiclaus) [reply drivers/rf-transceiver/ad9361/ad9361.c:490]: This was handled by the code style script. I will try to adjust them manually

Comment (amiclaus) [reply drivers/rf-transceiver/ad9361/ad9361.c:490]: I tried rearranging them in different manners, the indenting is kept the same (first line indented with one tab less) after applying the code style. Either we discard this change and the build will fail, either we keep this change as it is.

**Review by amiclaus — COMMENTED** [review]
V2:
- perform rebase

File: drivers/rf-transceiver/ad9361/ad9361.c (hunk 1 above)
Comment (mchindri) [reply drivers/rf-transceiver/ad9361/ad9361.c:490]: Is this needed? Coding style is ignored on master.

Comment (amiclaus) [reply drivers/rf-transceiver/ad9361/ad9361.c:490]: We can ignore it now, but as soon as this file will be changed again, we will face the same build issue at PR.

**Review by amiclaus — COMMENTED** [review]
V3:
- remove commit related to the code style issues detected. they will be handled in a different PR.

**Review by mchindri — APPROVED** [review]

**Review by scuciurean — APPROVED** [review]

**Pull request merged**